aat/src/aat/resources/features/F-081/S-347.td.json

```diff
@@ -0,0 +1,23 @@
+{
+  "_guid_": "S-347",
+  "_extends_": "F-081_Test_Base_Data",
+  "title": "must return 200 with List of jurisdictions",
+  "specs": [
+    "an active profile in CCD",
+    "contains a valid jurisdiction id",
+    "has the 200 OK code",
+    "contains the jurisdictions details for jurisdiction id BEFTA_CASETYPE_2_1"
+  ],
+  "expectedResponse": {
+    "_extends_" : "Common_200_Response",
+    "headers": {
+      "Content-Encoding" : "gzip",
+      "Content-Length" : "[[ANYTHING_PRESENT]]",
+      "Content-Type" : "application/json;charset=UTF-8",
+      "Strict-Transport-Security": "[[ANY_STRING_NULLABLE]]"
+    },
+    "body" : {
+      "arrayInMap" : "[[ANYTHING_PRESENT]]"
```

Comment:
If we are getting a specific jurisdiction, why are we not specifying its details inside the array??

Reply:
the response does not have a consistent ordering to enable us to have a more defined expected response

Reply:
hence i have linked to a ticket to rectify in the feature file

Reply:
but this one is returning only 1 jurisdiction object in the array, isn't it?

Reply:
been able to be more specific with the returned fields, for case type however there is no ordering in which case type is returned first